metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/meter/os/NetMeter.java

```diff
@@ -56,32 +60,55 @@
 
     private Id receiveId;
 
+    private Id sendPacketId;
```

Comment:
same to cpu
logic of send and receive is same, except value of id/read from index is different, extract to one class,and InterfaceInfo  have 2 field of it.